src/java/org/apache/cassandra/db/filter/ColumnFilter.java

```diff
@@ -385,44 +404,24 @@ public boolean equals(Object other)
     @Override
     public String toString()
     {
-        if (isFetchAll)
-            return "*";
-
-        if (queried.isEmpty())
-            return "";
-
-        Iterator<ColumnDefinition> defs = queried.selectOrderIterator();
-        if (!defs.hasNext())
-            return "<none>";
-
-        StringBuilder sb = new StringBuilder();
-        while (defs.hasNext())
-        {
-            appendColumnDef(sb, defs.next());
-            if (defs.hasNext())
-                sb.append(", ");
+        if (isFetchAll && queried == null)
+            return "*/*";
```

Comment:
Since you are changing the toString method already, maybe make it a bit more descriptive? Is the first `*` for fetched and the second for `queried`?

Reply:
Is there any code or test depends on the column filter toString output?

Reply:
I hope there is none. If there is such code, I will refactor it but we will know that once we run tests. Also I'm ok with making it more descriptive, however now it seem like it includes all the information - yes, the first star is for fetched and the second is for queried - in the previous version some information was lost (it is also more refactored in `trunk`)

Reply:
I also like the idea of making it more descriptive, showing which part is the queried one and which is the fetched one, maybe something like `{queried=*, fetched=[v2, v3]}`? I'm not sure whether it would be a too invasive change in the logs, though.

Reply:
Well, I think that information is not useful from the user pov. It is probably useful for developers trying to fix a bug and in such a case, they will easily decode the meaning of pre slash and post slash parts. The idea here was to make that method provide the string representation which is 1:1 mapped to the data in column filter. Previously we were loosing some information